documents/motions/09.2019.1 - Senior Delegates.md

```diff
@@ -28,11 +28,11 @@ Senior Delegates are WCA Staff and Full Delegates, with an extra role to manage
       3. Support Full Delegates, Candidate Delegates, and potential future WCA Delegates in their evolution, learning, and professionalisation.
```

Comment:
```suggestion
      3. Support Full Delegates, Candidate Delegates, and potential future WCA Delegates in their evolution, learning, and professionalization.
```

Reply:
Ups! I'll force-push an amended version of the PR right now.